### DISCUSSION TITLE
Integration to .net webForm

### DISCUSSION THREAD

**Discussion by vineetavenugopal** [optimajet/WorkflowEngine.NET#214] [Q&A]
I have an application ready where the requests will be added for the workflow . I would like to know can i integrate the workflow to my application and start the workflow on event of new requests added by button click.

**Comment by optimajet** [comment]
Hello @vineetavenugopal,

WorkflowEngine is specifically designed to be embedded into an existing application.

You can read the documentation here - https://workflowengine.io/documentation.
About integration - https://workflowengine.io/documentation/how-to-integrate.
Also check out our tutorials - https://workflowengine.io/documentation/tutorials.

**Comment by vineetavenugopal** [comment]
i have gone through these .... but these are integrating the designer to the .net  application..My requirement is to start the workflow from my application .

**Reply by optimajet** [reply]
What programming language is your application written in?
If you are not using .NET, there is Worklfow Server, where processes can be run via the HTTP API (https://workflowserver.io)?

**Reply by vineetavenugopal** [reply]
.net web forms with ms sql

**Reply by optimajet** [reply]
Steps 2 and 3 about the backend:
- https://workflowengine.io/documentation/how-to-integrate#database
- https://workflowengine.io/documentation/how-to-integrate#runtime

And here's another example - https://github.com/optimajet/WorkflowEngine.NET/tree/master/Samples/ASP.NET%20WebForms/WebFormsMSSQL
